src/qt/glcanvas.cpp

```diff
@@ -64,6 +64,259 @@ void wxQtGLWidget::paintGL()
     EmitEvent(event);
 }
 
+// ----------------------------------------------------------------------------
+// wxGLContextAttrs: OpenGL rendering context attributes
+// ----------------------------------------------------------------------------
+// GLX specific values
+
+wxGLContextAttrs& wxGLContextAttrs::CoreProfile()
+{
+//    AddAttribBits(GLX_CONTEXT_PROFILE_MASK_ARB,
+//                  GLX_CONTEXT_CORE_PROFILE_BIT_ARB);
+    SetNeedsARB();
+    return *this;
+}
+
+wxGLContextAttrs& wxGLContextAttrs::MajorVersion(int val)
+{
+    if ( val > 0 )
```

Comment:
This if() is not really needed, is it? I guess it was intended to be more like a "standard" condition, but it's actually confusing (i.e. was there some other code that was forgotten?).
The same goes for other [empty] if() blocks throughout the entire change.